cmd/kubelet/app/server.go

```diff
@@ -338,6 +226,35 @@ func validateConfig(s *options.KubeletServer) error {
 	return nil
 }
 
+// If successful, returns the configuration that should be used and the nodeconfig controller.
+// Otherwise returns an error.
+func initNodeConfig(bootstrapKC *componentconfigv1alpha1.KubeletConfiguration,
```

Comment:
would prefer a single path passed in, do the `if !isAbs(path) { path = filepath.join(root,path) }` logic before calling into here